projects/client-api-react/src/stories/Graphistry.stories.jsx

```diff
@@ -13,9 +13,11 @@ export const Empty = (args) => <Graphistry {...args} />;
 export const PredefinedDataset = (args) => <Graphistry {...args} dataset='Miserables' showSplashScreen={true} />;
 
 const defaultSettings = {
+  // graphistryHost: "http://0.0.0.0:3000",
   dataset: 'Miserables',
   play: 1,
-  showSplashScreen: true
+  showSplashScreen: true,
```

Comment:
do the `onXUpdate=undefined` here so works for all?

Reply:
yes.